gui/melt-remainder.lua

```diff
@@ -0,0 +1,119 @@
+-- Add a display for how much melted metal is stored inside smelters.
+--@module = true
+--@enable = true
+--[====[
+
+gui/melt-remainder
+==================
+When enabled, a line of text is added to the Tasks screen of melters and magma smelters
+that shows how much metal is "stored" as a result of melting items in it. (The base game
+has no display of this information,)
+
+Click on the text for more detailed information.
+
+]====]
+
+local overlay = require("plugins.overlay")
+local widgets = require("gui.widgets")
+-- the existence of this is not documented anywhere of course
+local dialogs = require("gui.dialogs")
+
+ENABLED = ENABLED or false
+
+local function i_hate_lua(tbl)
```

Comment:
This isn't really an appropriate place to express your opinions on the scripting language DFHack uses.